src/metabase/sync_database/sync.clj

```diff
@@ -143,6 +143,67 @@
       (catch Throwable t
         (log/error (u/format-color 'red "Unexpected error syncing table") t)))))
 
+(def ^:private ^:const blacklisted-table-names
```

Comment:
Since it's not really important why a table is in the list from a logic standpoint I think it makes sense to keep a single list and just to put the reasons for inclusion in comments rather than the code itself. We can still break up the list into groups by judicious use of comments and newlines where appropriate.